### ISSUE TITLE
[Idea] Update the Forge version for 1.19.2

### ISSUE THREAD

**Issue by boban97** [MCCTeam/Minecraft-Console-Client#2309]
### Prerequisites

- [X] I have read and understood the [user manual](https://github.com/MCCTeam/Minecraft-Console-Client/tree/master/MinecraftClient/config)
- [X] I made sure I am running the latest [development build](https://ci.appveyor.com/project/ORelio/Minecraft-Console-Client/build/artifacts)
- [X] I tried to [look for similar feature requests](https://github.com/MCCTeam/Minecraft-Console-Client/issues?q=is%3Aissue) before opening a new one

### Console Client Version

20221027-131

### Describe your problem

Can't connect to the latest (1.19.2) Forge server with the latest MCC version.
Can you update the protocol?

```ini
[MCC] Login failed :
[MCC] This modded server is not impl compatible with your modded client. Please verify your Forge version closely matches the server. Got net version 2 this server is net version 3
[ERROR] Failed to login to this server.
```

### Attach screenshot here (If applicable)

![mccerror](https://user-images.githubusercontent.com/44925672/198371904-b6794895-32ad-420e-946c-1101a7fb68f2.png)

### Minecraft Version

1.19.2

### Device

Desktop

### Operating System

Windows

**Comment by DasRudeltier** [comment]
Hello :)

Any changes yet?

**Comment by breadbyte** [comment]
Our focus right now is maintaining vanilla compatibility, we don't have enough hands to work on forge unfortunately.

**Comment by milutinke** [comment]
Implemented in #2644 , will be merged soon.
